cob_helper_tools/scripts/auto_recover.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python
+
```

Comment:
maybe we should start again to put [the headers](https://github.com/ipa320/setup/tree/master/templates) in our sourcefiles.